### PR TITLE
Add 3656666.xyz and forks.my.id

### PR DESCRIPTION
**Pull request by xxecy-114514**
# Public Suffix List (PSL) Submission

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (`make test`)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the `_psl` TXT record in place in the respective zone(s).

__Submitter affirms the following:__ 

 * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)
 - [Cloudflare](https://developers.cloudflare.com/learning-paths/get-started/add-domain-to-cf/add-site/)
 - [Let's Encrypt](https://letsencrypt.org/docs/rate-limits/)
 - MAKE SURE UPDATE THE FOLLOWING LIST WITH YOUR LIMITATIONS! REMOVE ENTRIES WHICH DO NOT APPLY AS WELL AS REMOVING THIS LINE!

 * [x] This request was _not_ submitted with the objective of working around other third-party limits.

 * [x] The submitter acknowledges that it is their responsibility to maintain the domains within their section. This includes removing names which are no longer used, retaining the _psl DNS entry, and responding to e-mails to the supplied address. Failure to maintain entries may result in removal of individual entries or the entire section.

 * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms to them.
 * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting.

---

For PRIVATE section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*. I could break my organization's website cookies and cause other issues, and the rollback timing is acceptable. *Proceed anyways*.
---

## Description of Organization

**Organization Website:**
**[link](https://nic.3656666.xyz)**

The "365 Free Domain " sub -domain registration platform is a method for providing a free network image for individuals, non -profit organizations and projects. We noticed that the domain name fee is gradually rising, which is very expensive for students and non -profit organizations. Therefore, the platform is created so that everyone can show their own website.

Inclusion of 3656666.xyz and forks.my.id in the Public Suffix List will benefit individuals and non-profit organizations that use these domains, especially in terms of cookie security.

We have carefully considered the potential impact of this inclusion and understand the responsibilities involved.

I kindly request that our PR review and consider inclusion of our domains in the Public Suffix List. Thank you for your interest and support.

## Reason for PSL Inclusion

`Whois`
image
image

We estimate that the (real) number of users will reach 200 by 2025-01-01

**Number of users this request is being made to serve:** 17（According to the public voting results）
image

The reasons why we hope to be included in PSL are as follows:

1. We hope that joining PSL can solve the problem of cookie security: Including the domain name of '365 Free Domain' in the PSL is critical to maintaining cookie security. By setting up a separate subdomain for each nonprofit, we prevent cookies from being shared between applications from different organizations. This isolation enhances the security and privacy of user data, ensuring that sensitive information remains in the appropriate organizational context.
2. We hope more people will use it: We noticed that the domain names included in PSL are used by a lot of people.

## DNS Verification

```
dig +short TXT _psl.3656666.xyz
"https://github.com/publicsuffix/list/pull/2193"
```

```
dig +short TXT _psl.forks.my.id
"https://github.com/publicsuffix/list/pull/2193"
```

## Results of Syntax Checker (`make test`)

```
PASS: test-is-public-builtin
PASS: test-is-public
PASS: test-is-cookie-domain-acceptable
PASS: test-registrable-domain
PASS: test-is-public-all

### CODE REVIEW

File: public_suffix_list.dat [hunk 1]
```diff
@@ -15738,4 +15743,6 @@ basicserver.io
 virtualserver.io
 enterprisecloud.nu
 
+
```
Comment (groundcat) [inline public_suffix_list.dat:15746]: Please do not add blank lines here or make any changes to parts of the file outside of the proposed block.

**Comment by wdhdev** [comment]
You are missing half the PR template. Please update it with your rationale and org info/desc.

**Comment by wdhdev** [comment]
@xxecy-114514 This project will unfortunately not be able to be approved at this point in time as you have zero users: 
- https://github.com/xxecy-114514/365-Free-Domain/blob/main/db.3656666.xyz
- https://github.com/xxecy-114514/365-Free-Domain/blob/main/db.forks.my.id

From the guidelines:

> Projects that are smaller in scale or are temporary or seasonal in nature will likely be declined. Examples of this might be private-use, sandbox, test, lab, beta, or other exploratory nature changes or requets. It should be expected that despite whatever site or service referred a requestor to seek addition of their domain(s) to the list, projects not serving more then thousands of users are quite likely to be declined.

**Comment by groundcat** [comment]
@xxecy-114514 Did you create this post?

https://linux.do/t/topic/221506 ([archive](https://web.archive.org/web/20241002011543/https://linux.do/t/topic/221506))

English translation:

> ## Hurry and register for a free domain here to help reach 200 people for the PSL list, and strive to get a free domain that can be bound to Cloudflare (CF).  
> Application link: hxxps://nic.3656666[.]xyz/  
> Try to bind the domain to serv00 and set up a service, or use another virtual hosting provider.  
> Here's one option: Byet reseller  
> Install WordPress or phpBB and write some random content.  
> We need about 200 people to apply. Everyone, let's make an effort and try to get a free domain that can be bound to Cloudflare.

Based on this, I'm afraid your project is trying to add the domain to the PSL for the sole purpose of bypassing third-party limits, which is against the [Validation and Non-Acceptance Factors](https://github.com/publicsuffix/list/wiki/Guidelines#validation-and-non-acceptance-factors) outlined in the guidelines.

**Comment by xxecy-114514** [comment]
> @xxecy-114514 Did you create this post?
> 
> https://linux.do/t/topic/221506 ([archive](https://web.archive.org/web/20241002011543/https://linux.do/t/topic/221506))
> 
> English translation:
> 
> > ## Hurry and register for a free domain here to help reach 200 people for the PSL list, and strive to get a free domain that can be bound to Cloudflare (CF).
> > Application link: hxxps://nic.3656666[.]xyz/
> > Try to bind the domain to serv00 and set up a service, or use another virtual hosting provider.
> > Here's one option: Byet reseller
> > Install WordPress or phpBB and write some random content.
> > We need about 200 people to apply. Everyone, let's make an effort and try to get a free domain that can be bound to Cloudflare.
> 
> Based on this, I'm under the impression that your project is trying to add the domain to the PSL for the sole purpose of bypassing third-party limits, which is against the [Validation and Non-Acceptance Factors](https://github.com/publicsuffix/list/wiki/Guidelines#validation-and-non-acceptance-factors) outlined in the guidelines.



> @xxecy-114514 This project will unfortunately not be able to be approved at this point in time as you have zero users:
> 
> * https://github.com/xxecy-114514/365-Free-Domain/blob/main/db.3656666.xyz
> * https://github.com/xxecy-114514/365-Free-Domain/blob/main/db.forks.my.id
> 
> From the guidelines:
> 
> > Projects that are smaller in scale or are temporary or seasonal in nature will likely be declined. Examples of this might be private-use, sandbox, test, lab, beta, or other exploratory nature changes or requets. It should be expected that despite whatever site or service referred a requestor to seek addition of their domain(s) to the list, projects not serving more then thousands of users are quite likely to be declined.

Currently it has zero users, we expect to reach 200 users by 2025-01-01

**Comment by xxecy-114514** [comment]
> @xxecy-114514 Did you create this post?
> 
> https://linux.do/t/topic/221506 ([archive](https://web.archive.org/web/20241002011543/https://linux.do/t/topic/221506))
> 
> English translation:
> 
> > ## Hurry and register for a free domain here to help reach 200 people for the PSL list, and strive to get a free domain that can be bound to Cloudflare (CF).
> > Application link: hxxps://nic.3656666[.]xyz/
> > Try to bind the domain to serv00 and set up a service, or use another virtual hosting provider.
> > Here's one option: Byet reseller
> > Install WordPress or phpBB and write some random content.
> > We need about 200 people to apply. Everyone, let's make an effort and try to get a free domain that can be bound to Cloudflare.
> 
> Based on this, I'm under the impression that your project is trying to add the domain to the PSL for the sole purpose of bypassing third-party limits, which is against the [Validation and Non-Acceptance Factors](https://github.com/publicsuffix/list/wiki/Guidelines#validation-and-non-acceptance-factors) outlined in the guidelines.

I just want to make this a publicity, because I found that the subdomains that can be bound by cloudflare are used by a lot of people. If you can tell me how to contact clouflare to bypass the restriction, I will close this PR

**Comment by wdhdev** [comment]
> Currently it has zero users, we expect to reach 200 users by 2025-01-01

That is still not eligible for PSL addition. We tend to require 1k+ users for approval.

> If you can tell me how to contact clouflare to bypass the restriction, I will close this PR

We are not associated with Cloudflare, so we cannot. It is evident that this PR is purely created to bypass limits Cloudflare has put in place for a reason, which is against our guidelines:

> We do not accept entries that have the objective of getting around limitations that have been put in place by a vendor to protect internet users. The PSL is not a 'workaround', and Pull Requests that appear to be doing this should expect to be declined. Be thorough and candid with the rationale furnished with the request.

You can contact Cloudflare yourself to attempt to remove this restriction, which I doubt they will.

**Pull request closed without merging**